tests/features/add_service.feature

```diff
@@ -0,0 +1,27 @@
+@api @javascript @multidev @dev
+Feature: Add a service feature
+  In order to manage services
+  As a group member
+  I need to be able to add a service
+
+  Scenario: Access service creation page and have stuff to do
+    Given I am logged in as user "marty.member@portlandoregon.gov"
+    When I visit "/group/14/content/create/group_node%3Acity_service"
+    Then I should see "Title"
+    And I should see "Step title"
+    And I should see "Step instruction"
+    And I should see "Related content"
+    And I should see "Legacy path"
+
+    When I press "Add Step"
+    And I wait for AJAX to finish
+    Then I should see "Step"
+
+    When I fill in "Title" with "Test service"
+    And I select "Online" from "edit-field-service-mode-0-subform-field-service-modes"
+    And I fill in "edit-revision-log-0-value" with:
```

Comment:
Is this using the new revision field that @jookyg added in POWR-444? If not, I think this could be problematic.